### PR TITLE
Feature/block level text align

### PR DESCRIPTION
**Pull request by andersryanc**

### CODE REVIEW

**Comment by Tellisense** [comment]
@andersryanc thanks Ryan, hopefully @sstur can review and merge this feature in!

**Comment by heyxmay** [comment]
Hey @sstur can you review and merge this feature in? It'll be really useful in my pet project!

**Comment by heyxmg** [comment]
Hey @sstur can you review and merge this feature in? It'll be really useful in my pet project!

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "build-lib": "rimraf lib && babel src --ignore \"_*\" --out-dir lib --copy-files",
     "lint": "eslint --max-warnings 0 .",
     "typecheck": "flow",
-    "prepublish": "npm run build",
+    "prepare": "npm run build",
```
Comment (sstur) [inline package.json:16]: 👍💯

**Comment by andersryanc** [comment]
> Wow, this is really awesome work @andersryanc! Thanks for the PR.

Thanks! Happy to see it getting merged!! 🎉

> The only thing I'd like to see is a screenshot so I can know what the icons/toolbar looks like with this feature.

I posted a screenshot as my first comment 👆

**Comment by andersryanc** [comment]
Also, see the GIF I posted in #18

**Comment by sstur** [comment]
> I posted a screenshot as my first comment 👆

Oops, I didn't see that, so I edited the PR description with the screenshot just a few mins ago. All good now!

**Comment by serhiibartkiv** [comment]
Not possible to add BLOCK_ALIGNMENT_BUTTONS to toolbarConfig.
By another hand - BLOCK_ALIGNMENT_BUTTONS missed in type ToolbarConfig/GroupName

**Comment by zilkenberg22** [comment]
@andersryanc The text-value has the style `<p style="text-align: center">` but the styles are lost after RichTextEditor.createValueFromString(props.text, 'html'). Is there a way I can retain the styles from the saved text-values inside the editor?

**Comment by guettawin** [comment]
> @andersryanc The text-value has the style `<p style="text-align: center">` but the styles are lost after RichTextEditor.createValueFromString(props.text, 'html'). Is there a way I can retain the styles from the saved text-values inside the editor?

Do your alignment buttons work? If it works, throw off your config pls

**Pull request merged**